modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -225,7 +220,7 @@ def map_upstream_status(response_object)
         self.code = 'DOC202'
         self.detail = "Upstream status: #{response_object['errorMessage']}"
       else
-        log_message_to_sentry('Unknown status value from Central Mail API', :warning, status:)
+        Rails.logger.warn("Unknown status value from EMMS API. Guid: #{guid}. EMMS status: #{status}")
```

Comment:
Opportunity for: 
`Rails.logger.warn('Unknown status value from EMMS API.', { guid: guid, central_mail_status: status })`